packages/core/src/components/checkbox/checkbox.css

```diff
@@ -115,6 +116,15 @@
   width: var(--icon-size);
 }
 
+/* HELPER TEXT */
+.hds-checkbox__helper-text {
+  color: var(--helper-color-default);
+  display: block;
+  font-size: var(--fontsize-body-m);
+  margin-top: var(--spacing-2-xs);
+  flex-basis: 100%;
```

Comment:
Could we have a shared helper for the helper text, for example in the Core utils?
Then it could be imported and extended when needed.